src/integrationTest/integrationTestsUtilities.ts

```diff
@@ -10,11 +10,25 @@ import * as vscode from 'vscode'
 // java8.al2 image does a while to pull
 const LAMBDA_SESSION_TIMEOUT = 60000
 
+export const MAPPING_SCHEMA_TIMEOUT = 50000
```

Comment:
Do we really need separate timeouts? This is roughly the same as LAMBDA_SESSION_TIMEOUT , they are both big. Just rename LAMBDA_SESSION_TIMEOUT to `SLOW_TEST_TIMEOUT` or something like that.